src/topp/MapAlignerIdentification.cpp

```diff
@@ -158,11 +158,27 @@ class TOPPMapAlignerIdentification :
                          vector<TransformationDescription>& transformations,
                          Int reference_index)
   {
-    algorithm.align(data, transformations, reference_index);
-
     // find model parameters:
     Param model_params = getParam_().copy("model:", true);
     String model_type = model_params.getValue("type");
+
+    try
+    {
+      algorithm.align(data, transformations, reference_index);
+    }
+    catch (Exception::MissingInformation& err)
+    {
+      if (getFlag_("force"))
+      {
+        OPENMS_LOG_ERROR
```

Comment:
in PeptideIndexer we can select how to handle the error situation with some nice string lists.
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/ANALYSIS/ID/PeptideIndexing.cpp#L51
which might be more expressive

Reply:
Good to know, but I don't want to make this more complex than necessary at the moment.